packages/grid/package.json

```diff
@@ -0,0 +1,13 @@
+{
+ "name": "@starryui/grid",
+ "version": "1.0.0",
+ "types": "index.d.ts",
+ "description": "",
+ "main": "index.js",
+ "scripts": {
+  "test": "echo \"Error: no test specified\" && exit 1"
+ },
+ "keywords": [],
+ "author": "",
+ "license": "ISC"
```

Comment:
License is wrong, should match all other components

Reply:
Done

Reply:
✅ Whoops - that was my bad, I was thinking it would have the letters MIT but ISC is "functionally equivalent" to MIT https://en.wikipedia.org/wiki/ISC_license